doc/rfcs/00000-rfc-process.md

```diff
@@ -0,0 +1,150 @@
+# RFC: RFC process
+
+The request for comments (RFC) process is intended to provide a consistent and
+controlled path for new features to enter Node.js. The process helps find
+consensus among Node.js community and collaborators on substantial changes to
```

Comment:
I think consensus among the Node.js community which is very diverse in ideas and opinions is not really attainable for most features. I cannot name a single Node.js feature with consensus among the community :) 

Consensus among collaborators (who we also try to make sure is a diverse group of many opinions) is already a very high bar.

Reply:
I guess it depends on your definition of consensus. Which means that maybe I ought to define the term in the doc so it's clear how I mean it.